src/components/Cards.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'; 
+
+import Paper from '@material-ui/core/Paper'; 
+
+import data from '../resources/CardsData'; 
+
+import FlashCard from './Card'; 
+
+//? What will I need from Material UI? 
+// Paper, for the main page 
+//? How will the cards be done? 
+// I've used a third party library in the past that might be useful here, especially for 'study mode' - not sure how I'll implement a game yet 
```

Comment:
Turns out that Material UI and react-flip-card worked fine together. Will probably just continue using this approach because I like the effect.